<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="Start">
       <attribute name="title">
        <string>Главная</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTextEdit" name="Preview">
          <property name="font">
           <font>
            <family>Segoe UI Variable Small</family>
            <pointsize>22</pointsize>
           </font>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI Variable Small'; font-size:22pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Складской модуль&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Данное приложения является реализацией конткретной информационной системы. Оно позволяет вести учет поступления и выдачи товара. Заполнять  справочники организаций - владельцев складов, товаров, работников и накладных. Определять текущее состояние склада, а именно, количество свободных мест. &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Использование данного приложения подразумевает наличие базовых навыков владения компьютером, а также приминения базовой логики. &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:16pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:16pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:16pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:16pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;*Данное приложение является интеллектуальной собственностью автора. &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;**Строго для лиц на имеющих сенсорных и психоневрологических заболеваний, старше 12 лет&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Warehouse">
       <attribute name="title">
        <string>Склады</string>
       </attribute>
       <widget class="QTableView" name="wh_table">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>5</y>
          <width>670</width>
          <height>590</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="wh_update">
        <property name="geometry">
         <rect>
          <x>780</x>
          <y>550</y>
          <width>90</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Обновить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="wh_add">
        <property name="geometry">
         <rect>
          <x>780</x>
          <y>10</y>
          <width>90</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>685</x>
          <y>150</y>
          <width>280</width>
          <height>220</height>
         </rect>
        </property>
        <property name="title">
         <string>Текущий элемент</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="4" column="0">
          <widget class="QPushButton" name="wh_modify">
           <property name="text">
            <string>Изменить</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLineEdit" name="wh_owner_line"/>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="wh_remove">
           <property name="text">
            <string>Удалить</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QLineEdit" name="wh_space_line"/>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLineEdit" name="wh_addr_line"/>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLineEdit" name="wh_name_line">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="Owner">
       <attribute name="title">
        <string>Организации</string>
       </attribute>
       <widget class="QTableView" name="ow_table">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>5</y>
          <width>670</width>
          <height>590</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="ow_add">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>10</y>
          <width>180</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ow_update">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>550</y>
          <width>180</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Обновить</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>685</x>
          <y>150</y>
          <width>280</width>
          <height>220</height>
         </rect>
        </property>
        <property name="title">
         <string>Текущий элемент</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0" colspan="2">
          <widget class="QLineEdit" name="ow_name_line">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="dragEnabled">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLineEdit" name="ow_person_line"/>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLineEdit" name="ow_phone_line"/>
         </item>
         <item row="3" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="ow_modify">
           <property name="text">
            <string>Изменить</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="ow_remove">
           <property name="text">
            <string>Удалить</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="Stuff">
       <attribute name="title">
        <string>Персонал</string>
       </attribute>
       <widget class="QTableView" name="stuff_table">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>5</y>
          <width>670</width>
          <height>590</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="stuff_update">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>550</y>
          <width>180</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Обновить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="stuff_add">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>10</y>
          <width>180</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>685</x>
          <y>150</y>
          <width>280</width>
          <height>280</height>
         </rect>
        </property>
        <property name="title">
         <string>Текущий элемент</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="2" column="0" colspan="2">
          <widget class="QLineEdit" name="st_salary_line"/>
         </item>
         <item row="5" column="0">
          <widget class="QPushButton" name="stuff_modify">
           <property name="text">
            <string>Изменить</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLineEdit" name="st_job_line"/>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QLineEdit" name="st_name_line">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QPushButton" name="stuff_remove">
           <property name="text">
            <string>Удалить</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QLineEdit" name="st_phone_line"/>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLineEdit" name="st_wh_line">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>710</x>
          <y>440</y>
          <width>240</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Работодатель и зарплаты</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>480</y>
          <width>180</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Премия</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Receipt">
       <attribute name="title">
        <string>Накладные</string>
       </attribute>
       <widget class="QPushButton" name="rc_update">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>550</y>
          <width>180</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Обновить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="rc_in">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>10</y>
          <width>180</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Приходная</string>
        </property>
       </widget>
       <widget class="QPushButton" name="rc_out">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>50</y>
          <width>180</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Расходная</string>
        </property>
       </widget>
       <widget class="QTableView" name="rc_table">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>5</y>
          <width>670</width>
          <height>590</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Products">
       <attribute name="title">
        <string>Товары</string>
       </attribute>
       <widget class="QTableView" name="prod_table">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>5</y>
          <width>670</width>
          <height>590</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="prod_update">
        <property name="geometry">
         <rect>
          <x>780</x>
          <y>550</y>
          <width>180</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Обновить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="prod_is">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>440</y>
          <width>220</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>В наличие</string>
        </property>
       </widget>
       <widget class="QPushButton" name="prod_sort">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>490</y>
          <width>220</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Отсортировать по цене</string>
        </property>
       </widget>
       <widget class="QPushButton" name="prod_add">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>10</y>
          <width>220</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить наименование</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>685</x>
          <y>150</y>
          <width>280</width>
          <height>280</height>
         </rect>
        </property>
        <property name="title">
         <string>Текущий элемент</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="2" column="0" colspan="2">
          <widget class="QLineEdit" name="pr_sh_line">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="pr_remove">
           <property name="text">
            <string>Удалить</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QLineEdit" name="pr_des_line"/>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="pr_modify">
           <property name="text">
            <string>Изменить</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLineEdit" name="pr_name_line">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLineEdit" name="pr_price_line">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>685</x>
          <y>60</y>
          <width>300</width>
          <height>78</height>
         </rect>
        </property>
        <property name="title">
         <string>Узнать цену</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QLineEdit" name="pr_search_line"/>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pr_search">
           <property name="text">
            <string>Поиск</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Соедениться с базой данных</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
